README.md

```diff
@@ -19,7 +19,7 @@ This template uses transfer-learning from VGG16 ImageNet. Making it easy to try
 | --- | --- | --- |
 | BATCH_SIZE | int | Batch size. Default `32`. |
 | EPOCHS | int | Epoch number. This template applies "Early stopping". Default `50`. |
-| LEARNING_RATE | float | Learning rate. Need to be from `0.0` to `1.0`. Default `0.001`. |
+| LEARNING_RATE | float | Learning rate. Need to be from `0.0` to `1.0`. Default `0.0001`. |
```

Comment:
Change default value of `LEARNING_RATE` since a small learning rate may be good for a transfer learning.